examples/decoding/plot_decoding_spoc_CMC.py

```diff
@@ -82,3 +82,8 @@
 plt.legend()
 mne.viz.tight_layout()
 plt.show()
+
+# Plot the contributions to the detected components (i.e., the forward model)
```

Comment:
can you add a `######################` (79 char) line before this? It will break it into sections so that the plot shows up after the code, which will render a bit nicer than this

https://10444-1301584-gh.circle-artifacts.com/0/html/auto_examples/decoding/plot_decoding_spoc_CMC.html

And I agree that the sources are not motor cortex :)